cli/src/args.rs

```diff
@@ -19,12 +19,18 @@ pub const ARG_OUTPUT_FOLDER: &str = "output-folder";
 pub const ARG_FOLLOW_LINKS: &str = "follow-links";
 pub const ARG_TARGET_OS: &str = "target_os";
 
-#[cfg(feature = "go")]
+#[cfg(all(feature = "go", feature = "python"))]
 const AVAILABLE_LANGUAGES: [&str; 6] = ["kotlin", "scala", "swift", "typescript", "go", "python"];
 
-#[cfg(not(feature = "go"))]
+#[cfg(all(feature = "go", not(feature = "python")))]
+const AVAILABLE_LANGUAGES: [&str; 5] = ["kotlin", "scala", "swift", "typescript", "go"];
+
+#[cfg(all(feature = "python", not(feature = "go")))]
 const AVAILABLE_LANGUAGES: [&str; 5] = ["kotlin", "scala", "swift", "typescript", "python"];
 
+#[cfg(not(any(feature = "go", feature = "python")))]
+const AVAILABLE_LANGUAGES: [&str; 4] = ["kotlin", "scala", "swift", "typescript"];
+
```

Comment:
It becomes rather convoluted to have to maintain 4 lists, just for the benefit of having this be a const. Since this is only used in the function below (`build_command`) let's instantiate it as a mutable slice within the function and add go and python to it based on the active features.

WDYT?

Reply:
Wouldn't it be the something similar as we would need to check for which feature is currently active? I don't think Rust has a built in feature list on compilation. We would still need to do something like so:
```
pub(crate) fn build_command() -> Command<'static> {
let  languages = vec!["kotlin", "scala", "swift", "typescript"]
let available_languages: &mut [&str] = &mut languages[..]
#[cfg(not(any(feature = "go", feature = "python")))]
logic
#[cfg(all(feature = "python", not(feature = "go")))]
logic
``` 
etc..

I don't mind it like this as `build_command` is purely generating the available options to it and adding logic to it based off available features would convolute that function.

Reply:
This scales exponentially.

If we add a new language, there would be 8 lists to maintain. If we add another one, 16.

This can be more elegantly modelled like:
```
    let  mut languages = vec!["kotlin", "scala", "swift", "typescript"];
    #[cfg(feature = "go")]
    languages.push("go");
    #[cfg(feature = "python")]
    languages.push("python");
```

Reply:
Ahh I see that is much nicer! Done